cfme/control/__init__.py

```diff
@@ -1,4 +1,4 @@
-from . import explorer  # noqa: F401
-from . import import_export  # noqa: F401
-from . import log  # noqa: F401
-from . import simulation  # noqa: F401
+from cfme.control import explorer  # noqa: F401
```

Comment:
I don't understand the use of importing these. Why can't we just leave the `__init__.py` empty?

Reply:
This allows for import of these modules though `cfme.control`.  If you would like to remove them I would certainly entertain that PR, but you'll have to identify where these imports are used in other modules and update them accordingly. I would not like to include that change in this PR.